augur/tasks/start_tasks.py

```diff
@@ -365,14 +365,14 @@ def augur_collection_monitor():
         enabled_phase_names = get_enabled_phase_names_from_config(session.logger, session)
 
         if primary_repo_collect_phase.__name__ in enabled_phase_names:
```

Comment:
In the future we should get the amount of repos to schedule from the amount of processes we have available and then divide them among the collection hooks. This way one repo gets one process. For now we can just hard code it I think.